### PR TITLE
Resolve #1289 Updated alert styles with the native component

### LINKED ISSUE #1289: Change all alerts style to Material UI 4.9

### PR DESCRIPTION
**Pull request by williamleha**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -11,8 +11,6 @@
     <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicon-32x32.png" />
     <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicon-16x16.png" />
     <link rel="mask-icon" href="%PUBLIC_URL%/safari-pinned-tab.svg" color="#000" />
-    <!-- <meta name="msapplication-TileColor" content="#ffffff" />
```
Comment (timothywangdev) [inline public/index.html]: why change this

File: src/components/AddAccountModalComponent.jsx [hunk 2]
```diff
@@ -693,7 +681,7 @@ const AddAccountModalComponent = (props: AddAccountModalComponentProps) => {
     }
   }, [actionsFulfilled['newCryptoAccountsFromWallet']])
 
-  function renderSteps () {
+  function renderSteps() {
```
Comment (timothywangdev) [inline src/components/AddAccountModalComponent.jsx]: space between

File: src/components/ReceiptComponent.jsx [hunk 3]
```diff
@@ -52,7 +53,7 @@ class ReceiptComponent extends Component<Props, State> {
     })
   }
 
-  renderReceipt () {
+  renderReceipt() {
```
Comment (timothywangdev) [inline src/components/ReceiptComponent.jsx]: space between

File: src/components/ReceiptComponent.jsx [hunk 4]
```diff
@@ -562,7 +565,7 @@ class ReceiptComponent extends Component<Props, State> {
       )
   }
 
-  render () {
+  render() {
```
Comment (timothywangdev) [inline src/components/ReceiptComponent.jsx]: space between

File: src/components/ReceiveFormComponent.jsx [hunk 5]
```diff
@@ -343,7 +330,7 @@ class ReceiveFormComponent extends Component {
     password: ''
   }
 
-  componentDidMount () {
+  componentDidMount() {
```
Comment (timothywangdev) [inline src/components/ReceiveFormComponent.jsx]: space between

File: src/components/ReceiveFormComponent.jsx [hunk 6]
```diff
@@ -368,7 +355,7 @@ class ReceiveFormComponent extends Component {
     }
   }
 
-  renderUnableToAccept () {
```
Comment (timothywangdev) [inline src/components/ReceiveFormComponent.jsx]: space between

File: src/components/ReceiveFormComponent.jsx [hunk 7]
```diff
@@ -368,7 +355,7 @@ class ReceiveFormComponent extends Component {
...
+  renderUnableToAccept() {
```
Comment (timothywangdev) [inline src/components/ReceiveFormComponent.jsx]: space between

File: src/components/ReceiveFormComponent.jsx [hunk 8]
```diff
@@ -410,7 +397,7 @@ class ReceiveFormComponent extends Component {
     )
   }
 
-  render () {
+  render() {
```
Comment (timothywangdev) [inline src/components/ReceiveFormComponent.jsx]: space between

File: public/index.html (hunk 1 above)
Comment (williamleha) [reply public/index.html]: Windows 8, 10 的favicon好像走browserconfig.xml就行了
https://realfavicongenerator.net/favicon_checker?protocol=https&site=lezhong.serveo.ventureum.io%2F#.XnvgFZNKgWr

Comment (timothywangdev) [reply public/index.html]: remove this if not needed

File: src/components/AddAccountModalComponent.jsx [hunk 9]
```diff
@@ -676,7 +664,7 @@ const AddAccountModalComponent = (props: AddAccountModalComponentProps) => {
     if (cryptoType) setPlateformType(getCryptoPlatformType(cryptoType))
   }, [cryptoType])
 
-  function handleWalletSelect (walletType) {
+  function handleWalletSelect(walletType) {
```
Comment (timothywangdev) [inline src/components/AddAccountModalComponent.jsx]: space between

**Pull request merged**